packages/tea-map-cache/src/index.ts

```diff
@@ -1,6 +1,7 @@
 /* eslint-disable no-console */
-const AWS = require('aws-sdk');
-const { getTeaBucketPath } = require('./tea');
+import AWS from 'aws-sdk';
```

Comment:
Is this eslint-disable line still relevant?

Also, out of curiosity, why are we using the AWS SDK directly and not aws-client? It doesn't really matter, but how does AWS.S3() know to use the correct region?

Reply:
.... that is an excellent question.     And might impact the release.    

🤔

Reply:
The AWS lambda runtime assumes same-region access for the lambda: 

https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html

as such, it should have AWS_REGION set to whatever the lambda is deployed to, and Core assumes a *one region* deployment. 


That said, there's no good reason not to use the aws-client for it's better traces/etc.

Reply:
e7ea8b28a - nicely makes the mock a bit simpler too.